docs/Tutorials/Tutorial-design.md

```diff
@@ -0,0 +1,38 @@
+* Based on the needs of the community, and in consultation with other event planners, build an outline of desired learning outcomes.  
+* Consider how you would like to balance teaching on core concepts versus the technical aspects of operating a specific tool or library. Be mindful of spending too much time on a tool you might have just discovered that represents the latest and greatest, but might not be something the community decides to adopt in the long term. 
+* Include content that provides an initial picture of what might be possible when applying a particular data science tool. Save detailed explanations on a narrow topic for a later breakout session.
+* Start with simple explanations that attend to people who are seeing this content for the first time, AND keep advanced participants engaged by inviting them to assist others, or to explore more advanced concepts through individual study.
+* Make your tutorials interactive through live coding challenges, working through a Juypter Notebook together, or interactive discussions. Leave some space when you are not talking for people to work through examples and ask questions. 
+* take a bit of time to review tutorials offered at our previous hackweeks. [Here](https://www.youtube.com/watch?v=VMJ1ZO48cwU) is an example of a particularly well-crafted tutorial by Catherine Kuhn at the 2018 Geo Hack Week.
+
+## Tutorial Timeline
+
+It is a good idea to prepare your tutorials well in advance of the event so that they can be reviewed by other tutorial developers and members of the community. In some cases early preparation is required so that shared computing resources can be tested and configured to contain the libraries and datasets necessary to run the tutorials. Early tutorial preparation also cretes an opportunity for exploring the overall flow of tutorial ideas and concepts across the event. 
+
+## Technical Considerations
+
+Teaching software and data science tools to a large group of people is challenging when each person arrives with a different computer and operating system configuration. To avoid these challenges we often deploy shared computing resources, such as an instance of JupyterHub on a commercial cloud platform. We then have participants log in to this system using GitHub credentials. This centralized computing architecture can be directly linked to virtual drives that store any of the sample datasets used in the tutorials. Each tutorial can also be tested in advance on this system to ensure all the correct libraries and tools are installed.

```

Comment:
Is there a technical write-up of how you do this in practice? I feel like we struggle a lot with this at Astro Hack Week, and I think in general having an example of how a hack week solves this problem might be super valuable.

Reply:
putting in a few links for now. I agree a complete write-up would be useful!